nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-client-service/src/main/java/org/apache/nifi/distributed/cache/client/DistributedMapCacheClientService.java

```diff
@@ -99,411 +101,140 @@
     }
 
     @OnEnabled
-    public void cacheConfig(final ConfigurationContext context) {
-        this.configContext = context;
+    public void onEnabled(final ConfigurationContext context) {
+        super.enabled();
```

Comment:
It should not be necessary to invoke this method, the frameworks should handle invoking all annotated methods.